static/app/components/sidebar/isItemActive.tsx

```diff
@@ -0,0 +1,30 @@
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
```

Comment:
One small thought - idk if this warrants a separate file. I don't oppose it, but I would be totally fine if we just exported this fn from sidebarItem. Main reason to do that is that it's only used in the sidebar components right now and because isItemActive sounds kinda generic to me.

Reply:
Honestly, I agree with you! I _expected_ that people would ask me to split it out (some people seem to prefer that), but I'm happy to keep it in the same file

Reply:
I moved it back 👌🏻